src/client/api/v1/get_balance.h

```diff
@@ -11,6 +11,11 @@
 #include "core/address.h"
 #include "core/types.h"
 
+/**
+ * @brief Enum to specify the address type
+ */
+typedef enum { ED25519 = 1, BECH32 } addr_type_enum;
```

Comment:
this enum should decalar in `core/address.h`
to avoid conflict and confusion, please use meaningful names something like:
```suggestion
typedef enum { ED25519_ADDR = 1, BECH32_ADDR } addr_type_e;
```
why ES25519 set to `1`?

Reply:
As suggested the enum is removed and added bool is_bech32 parameter.
ES25519 was set to 1 just to avoid it beign set to default value 0.